commons/commons-serialization/src/testFixtures/kotlin/ac/uk/ebi/biostd/TestFunctions.kt

```diff
@@ -3,13 +3,15 @@ package ac.uk.ebi.biostd
 import ac.uk.ebi.biostd.common.SerializationConfig
 import ac.uk.ebi.biostd.integration.SubFormat
 import ebi.ac.uk.model.BioFile
+import kotlinx.coroutines.flow.asFlow
+import kotlinx.coroutines.runBlocking
 import java.io.File
 import java.nio.file.Files
 import kotlin.io.path.outputStream
 
-fun createFileList(vararg files: BioFile, format: SubFormat = SubFormat.JSON): File {
+fun createFileList(vararg files: BioFile, format: SubFormat = SubFormat.JSON): File = runBlocking {
     val file = Files.createTempFile("file-list", "${files.size}-files")
     val serializer = SerializationConfig.serializationService()
-    file.outputStream().use { serializer.serializeFileList(files.asSequence(), format, it) }
-    return file.toFile()
+    file.outputStream().use { serializer.serializeFileList(files.asFlow(), format, it) }
```

Comment:
Shouldn't this also be executed in the IO context?

Reply:
yes but it will require the function to be suspended and we use it as a top level. So because it is a test I think runBlocking will do.